src/Elcodi/Component/Menu/Entity/Menu/Node.php

```diff
@@ -290,6 +297,10 @@ public function isActive($currentUrl)
      */
     public function isExpanded($currentUrl)
     {
+        if (!($this->subnodes instanceof Collection)) {
```

Comment:
Are you sure of that?
first is `!$this->subnodes`
then `{result} instanceof Collection